test.js

```diff
@@ -1,96 +1,129 @@
+	try {
+		const content = fs.readFileSync(file, 'utf8')
+		return content
+	} catch (error) {
+		if (attempts >0 ) {
+			setTimeout(()=>{read(file,attempts -1)}, 500 )
+		}
+		throw error
+	}
 
-function write (file, data) {
-  return fs.appendFileSync(file, data, 'utf8')
 }
 
-function mkdir (dir) {
-  return fs.mkdirSync(dir)
+function write(file, data) {
+	return fs.appendFileSync(file, data, 'utf8')
+}
+/**
+ * Create directory structure
+ * @param {*} dir
+ * @param {*} options
+ */
+function mkdir(dir) {
+	fs.ensureDirSync(dir)
 }
 
-test.beforeEach(t => {
-  rimraf.sync('tmp')
-  fs.mkdirSync('tmp')
+test.beforeEach( () => {
+	fs.emptyDirSync('tmp')
+	fs.ensureDirSync('tmp')
 })
 
-test.afterEach(t => {
-  rimraf.sync('tmp')
+test.afterEach( () => {
+	fs.emptyDirSync('tmp')
 })
 
 /*
  *  === Single file or directory copy ===
  */
-test('copy one file to new name', t => {
-  fn('LICENSE', 'tmp/LICENSE')
-  t.is(read('LICENSE'), read('tmp/LICENSE'))
+test.serial('copy one file to new name', t => {
+	fn('LICENSE', 'tmp')
+	t.is(read('LICENSE'), read('tmp/LICENSE'))
 })
 
-test('copy one file to existing directory', t => {
-  mkdir('tmp/newdir')
-  fn('LICENSE', 'tmp/newdir')
-  t.is(read('LICENSE'), read('tmp/newdir/LICENSE'))
+test.serial('copy one file to existing directory', t => {
+	mkdir('tmp/newdir')
+	fn('LICENSE', 'tmp/newdir')
+	t.is(read('LICENSE'), read('tmp/newdir/LICENSE'))
 })
 
-test('copy one directory preserving file structure', t => {
-  mkdir('tmp/dir_old')
-  write('tmp/dir_old/file', 'mytext')
-  fn('tmp/dir_old', 'tmp/dir_new')
-  t.is(read('tmp/dir_new/file'), 'mytext')
+test.serial('copy one directory preserving file structure', t => {
+	debugger
+	mkdir('tmp/dir_old', { recursive: true } )
+	write('tmp/dir_old/file', 'mytext')
+	fn('tmp/dir_old', 'tmp/dir_new')
+	t.is(read('tmp/dir_new/file'), 'mytext')
 })
 
 /*
  *  === Multiple file or directory copy ===
  */
 
-test('copy multiple files and directories', t => {
-  mkdir('tmp/dir')
-  mkdir('tmp/dir/subdir')
-  write('tmp/dir/subdir/file', 'mytext')
+test.serial('copy multiple files and directories', t => {
+	mkdir('tmp/dir/subdir', { recursive: true })
+	write('tmp/dir/subdir/file', 'mytext')
+	fn(['LICENSE', 'package.json', 'tmp/dir/subdir'], 'tmp')
 
-  fn(['LICENSE', 'package.json', 'dir/subdir'], 'tmp')
-  t.is(read('LICENSE'), read('tmp/LICENSE'))
-  t.is(read('package.json'), read('tmp/package.json'))
-  t.is(read('tmp/dir/subdir/file'), 'mytext')
+	t.is(read( 'LICENSE'), read('./tmp/LICENSE'),  'LICENSE File incorrect')
+	t.is(read('package.json'), read('./tmp/package.json'), 'tmp/package.json File incorrect')
+	t.is(read('tmp/dir/subdir/file'), 'mytext', 'tmp/dir/subdir/file File incorrect')
 })
 
 /*
  *  === Non-existant directory copy ===
  */
-// test('copy one file into non-existing directory', t => {
-//   fn('LICENSE', 'tmp/dir_nonexist/newfile')
-// })
+
+test.serial('copy one file into non-existing directory', t => {
```

Comment:
I'm thinking that hound is something new?  The current tests use arrow syntax, so the existing code would fail this review?  

As the existing tests are using this syntax, I think that this review bot hasn't been set up to use ES6 - will investigate (use 'esversion: 6') as identified in the review.

Reply:
Yes, between new and suspicious, the same silly errors...
I wonder if I can delete this spam... please ignore and sorry about that.